src/tests/unit/tests/common/components/cards/card-kebab-menu-button.test.tsx

```diff
@@ -144,6 +144,22 @@ describe('CardKebabMenuButtonTest', () => {
         expect(rendered.find(ActionButton).prop('menuProps')).toMatchSnapshot();
     });
 
+    it.each(['test-kebabmenuarialabel', undefined])(
+        'renders per snapshot with allCardInteractionsSupported and %s aria label passed as prop',
+        ariaLabel => {
+            const newProps = {
+                ...defaultProps,
+                kebabMenuAriaLabel: ariaLabel,
+            };
+            const rendered = shallow(
+                <CardKebabMenuButton {...newProps} deps={{ ...defaultDeps, cardInteractionSupport: allCardInteractionsSupported }} />,
+            );
+
+            expect(rendered.debug()).toMatchSnapshot();
```

Comment:
why `.debug()` instead of `.getElement()`?

Reply:
`debug` gave me more information than `getElement` in this case; so used that.